### DISCUSSION TITLE
Moving in observation rooms

### DISCUSSION THREAD

**Discussion by sparrowdragon** [BEEmod/BEE2-items#4292] [Feature Suggestions]
I'd like to suggest a feature so that if players noclip into ANY observation room, they can turn off noclip and move around in them without invisible barriers. Currently, you can't move around in some, here are a few examples : (probably) ANY half-block observation room, small wall rooms in P1 and Overgrown, large ones in Antique themes...

**Comment by TeamSpen210** [comment]
You're not really supposed to be able to get into observation rooms, so they've been optimised for viewing from outside. You'll notice invisible surfaces, missing details, the glass being one-way, etc. The barrier brush is there to make sure you can't portal or otherwise get into the room, which would be rather problematic. I'm not sure why you want to be in there anyway, there's nothing there.

**Reply by sparrowdragon** [reply]
I know you're not actually supposed to. But I think it's fun, and as I mentioned, it would be nice if I could move around in any observation room by turning Noclip on just to get there, then turn it back off. I know you can do that in some observation rooms in the Portal games and BEE-mod puzzles, as well as all the ones already available in the default P2 puzzlemaker. There doesn't really seem to be a point in having the barriers take up the entire observation room either, instead of just the windows and walls.

**Reply by FrozenDragon0** [reply]
Again, this is completely pointless and unnecessary, also you didn't read as the invisible barrier is there to ensure you can't portal into the room at all via glitches. Also if the barrier wasn't there and you do manage to glitch into the room what happens if you place both portals in there? Boom you've trapped yourself.

**Reply by sparrowdragon** [reply]
Okay, it's unnecessary, but I think you missed the part that said only **some** observation rooms had the barrier. Unless you're trying to say that those observation rooms that _have_ the extra barrier were originally more likely to cause the player to glitch into them by accident? Glitching intentionally, however, is a fun little trick when playing alone and/or showing it off in a video online.

**Reply by slambo311** [reply]
This is more of a Hammer thing than a PeTI/BEE2 thing. PeTI is for making actual tests, which this is very clearly not about.

**Comment by FrozenDragon0** [comment]
There's really no point in adding this as it's basically useless as Spencer mentioned.

**Reply by A-Job-Application** [reply]
Adding what? I can't help but notice that you've already added the barriers to at least some of the offices. So we wouldn't be adding anything, just deleting some useless, unrealistic matter that Valve themselves didn't care to add to the Editor OBS rooms.
